src/index.ts

```diff
@@ -1,4 +1,16 @@
 import { inspect } from "util";
+import fs from 'fs';
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
Yep, I agree. This specific part of code is probably the most important one

Reply:
very true